app/cp_nigeria/models.py

```diff
@@ -13,7 +8,7 @@ class Project(models.Model):
     latitude = models.FloatField()
     longitude = models.FloatField()
     date_start = models.DateTimeField()
-    date_end = models.DateTimeField()
```

Comment:
whenever you change something to the DB, you should then run `pyhton manage.py makemigrations` and also commit the migrations file (it must be linted with black before that).

I would rather keep `date_end` and modify the form widget to ask for duration instead and compute the end_date in the clean_end_date method of the form. One can even add the property method `duration` to the class `Project` which returns the difference between project-start and project end

Reply:
Lets make this change after PR 24, 19 and 23 are merged